TrafficCapture/trafficCaptureProxyServerTest/build.gradle

```diff
@@ -57,41 +57,44 @@ dockerFilesForExternalServices.each { projectName, dockerImageName ->
 }
 
 def javaContainerServices = [
-        ":TrafficCapture:trafficCaptureProxyServerTest": "jmeter"
+        "jmeter": ":TrafficCapture:trafficCaptureProxyServerTest"
 ]
```

Comment:
this is published under `migrations/jmeter`, can we keep this in this PR since this is an existing feature and we can make a separate one to review regarding removing it. I'd like to get more maintainers input on that.